packages/integration-tests/projects/connect-popup/index.js

```diff
@@ -0,0 +1,370 @@
+const puppeteer = require('puppeteer');
+const fs = require('fs');
+const path = require('path');
+
+const { Controller } = require('../../websocket-client');
+
+const defaults = {
+    // some random empty seed. most of the test don't need any account history so it is better not to slow them down with all all seed
+    mnemonic:
+        'alcohol woman abuse must during monitor noble actual mixed trade anger aisle',
+    pin: '',
+    passphrase_protection: false,
+    label: 'My Trevor',
+    needs_backup: false,
+};
+
+console.log(process.env);
```

Comment:
Why not using the `log` function that is defined below in this file so it is more consistent?

Reply:
gone.